### PR TITLE
Unary ops

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by fredrikekre** [comment]
Hmm... this happened on top of #58 review that and merge plz?

File: src/tensor_products.jl [hunk 1]
```diff
@@ -223,10 +248,35 @@ end
...
+ 0.566237  0.794026  0.854147
+
+julia> dot(A)
+3×3 Tensors.SymmetricTensor{2,3,Float64,6}:
+ 1.2577   1.25546  1.42706
+ 1.25546  1.43013  1.47772
+ 1.42706  1.47772  1.68067
+```
+"""
+@inline Base.dot(S::SymmetricTensor{2}) = tdot(S)
```
Comment (KristofferC) [inline src/tensor_products.jl]: I wonder if this `tdot` business is needed or if the compiler will optimize that stuff by itself.

**Comment by KristofferC** [comment]
Can you rebase this

File: src/tensor_products.jl (hunk 1 above)
Comment (fredrikekre) [inline src/tensor_products.jl]: What do you mean? `dot(S, S)` is a `Tensor`

Comment (KristofferC) [inline src/tensor_products.jl]: Oh yeah

**Pull request merged**